internal/pkg/earpointer/analysis_test.go

```diff
@@ -151,12 +151,10 @@ func TestField(t *testing.T) {
 	}
 	// t2's field x points to t3
 	want := concat(map[string]string{
-		"{f.t0}":         "--> *f.t0",
-		"{*f.t0}":        "[x->f.t1]",
-		"{f.t1}":         "--> f.a",
-		"{f.a}":          "[]",
-		"{f.t2}":         "[x->f.t3]",
-		"{f.t2[.],f.t3}": "--> f.a",
+		"{f.t0}":       "--> f.t2",
+		"{*f.t0,f.t2}": "[x->f.t3]",
+		"{f.t1,f.t3}":  "--> f.a",
```

Comment:
It feels a bit weird for `t1`and `t3` to be in the same partition, given that they have different types (`**int` and `*int`, respectively).